build.gradle.kts

```diff
@@ -1,14 +1,29 @@
 plugins {
   id("org.hypertrace.repository-plugin") version "0.2.1"
   id("org.hypertrace.ci-utils-plugin") version "0.1.2"
+  id("org.hypertrace.publish-plugin") version "0.3.0" apply false
   id("org.hypertrace.docker-java-application-plugin") version "0.7.1" apply false
   id("org.hypertrace.docker-publish-plugin") version "0.7.1" apply false
   id("org.hypertrace.jacoco-report-plugin") version "0.1.1" apply false
+  id("org.hypertrace.avro-plugin") version "0.2.4" apply false
 }
 
 subprojects {
   group = "org.hypertrace.ingester"
 
+  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
+    configure<org.hypertrace.gradle.publishing.HypertracePublishExtension> {
+      license.set(org.hypertrace.gradle.publishing.License.TRACEABLE_COMMUNITY)
+    }
+  }
+
+
+//  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
+//    configure<HypertracePublishExtension> {
+//      license.set(License.APACHE_2_0)
```

Comment:
we had one project here which was formerly ASL. @aaron-steinfeld do you happen to know the syntax to make an exception per-project? If so, feel free to push a commit to do that!

Reply:
something subtle about the following isn't working.. would be easier if this were property driven

```kotlin
subprojects.forEach {
  group = "org.hypertrace.ingester"

  pluginManager.withPlugin("org.hypertrace.publish-plugin") {
    // span-normalizer historically had a different license
    val licenseVal = if (it.projectDir.getName() == "span-normalizer")
      License.APACHE_2_0 else License.TRACEABLE_COMMUNITY

    configure<HypertracePublishExtension> {
      license.set(licenseVal)
    }
  }
```

Reply:
nm it was really only one module